src/lib.rs

```diff
@@ -0,0 +1,32 @@
+mod lines_to_params;
+mod parser;
+mod utils;
+
+use pyo3::prelude::*;
+use pyo3::types::PyDict;
+
+
+/// Main module for rust extensions with nari-act
+#[pymodule]
+fn nari_act_rust(py: Python, m: &PyModule) -> PyResult<()> {
+    m.add_function(wrap_pyfunction!(lines_to_params::ability_from_params, m)?)?;
+    m.add_function(wrap_pyfunction!(lines_to_params::action_effect_from_params, m)?)?;
+    m.add_function(wrap_pyfunction!(lines_to_params::status_effect_from_params, m)?)?;
+    m.add_function(wrap_pyfunction!(lines_to_params::statuslist_from_params, m)?)?;
+
+    let utils_mod = PyModule::new(py, "utils")?;
+    utils_mod.add_function(wrap_pyfunction!(utils::date_from_cs_string, m)?)?;
+    utils_mod.add_function(wrap_pyfunction!(utils::pad4, m)?)?;
+    utils_mod.add_function(wrap_pyfunction!(utils::pad8, m)?)?;
```

Comment:
Do these need to be exposed? It's fine if they do, I just don't see any use in python code.

Reply:
From what I recall, it was during my testing of zfill vs rust

Reply:
Probably best to expose what python actually uses then, rather than a subset of the method. Doesn't have to be included in this PR tho.